src/Configuration/Config/test/Microsoft.Extensions.Configuration.Tests.csproj

```diff
@@ -1,9 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <TargetFrameworks>$(StandardTestTfms);net461</TargetFrameworks>
```

Comment:
Just making sure this net461 is intentional

Reply:
Yes, because the AzureKeyVault tests target it, and they now depend on this for the base infrastructure.